test/Bin/GameTest/GameTests.cpp

```diff
@@ -1458,7 +1458,7 @@ GAME_TEST(Issues, Issue832) {
         return std::count_if(pActors.begin(), pActors.end(), [] (auto &&actor) { return actor.aiState == AIState::Dead; });
     });
     test->playTraceFromTestData("issue_832.mm7", "issue_832.json");
-    EXPECT_EQ(deathsTape.firstLast(), tape(0, 3));
+    EXPECT_EQ(deathsTape.firstLast(), tape(0, 4));
```

Comment:
Apparently one of the peasants far away now walks straight into a stray ice blast X)